src/App.css

```diff
@@ -2,4 +2,20 @@
 
 * {
     font-family: 'Montserrat', sans-serif;
+}
+
+.logo {
+  width: 150px;
+  height: 36px;
+  align-self: center;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: #636363;
+  font-size: 16px;
```

Comment:
You can add `margin-left: 26px;` here instead of the nth-child rule below. Making it 26px instead of 25px will make it pixel-perfect btw ;-)